tests/peft_integration/test_peft_pipelines_unnessesary_msg.py

```diff
@@ -0,0 +1,67 @@
+from transformers import (
+    AutoModelForCausalLM,
+    AutoTokenizer,
+    pipeline,
+)
+from transformers.testing_utils import (
+    require_peft,
+    require_torch,
+    slow,
+)
+from transformers.utils import is_torch_available
+
+
+@slow
+@require_torch
+@require_peft
+def test_peft_pipelines_unnessesary_msg():
+    if is_torch_available():
+        import torch
+
+    from peft import PeftModel
+
+    ADAPTER_PATH = "sajjadhadi/Disease-Diagnosis-Qwen2.5-0.5B"
+    BASE_PATH = "Qwen/Qwen2.5-0.5B"
```

Comment:
Let's use smaller models, from the hf internal repos:

```suggestion
    ADAPTER_PATH = "peft-internal-testing/tiny-OPTForCausalLM-lora"
    BASE_PATH = "hf-internal-testing/tiny-random-OPTForCausalLM"
```

Not sure if we would still require the `@slow` decorator in this case.